tests/test_pint_units.py

```diff
@@ -0,0 +1,39 @@
+import pint
+import pytest
+
+from pnpq.units import ureg
+
+
+@pytest.fixture(name="custom_ureg")
+def ureg_fixture() -> pint.UnitRegistry:
+    return ureg
```

Comment:
I think we don't need this fixture. Using the imported `ureg` directly within each function should work just fine.